### ISSUE TITLE
Restrict CORS rules on the Python server

### ISSUE THREAD

**Issue by milancurcic** [fortran-lang/playground#34]
I didn't notice this before but Flask's CORS defaults, i.e.:

https://github.com/fortran-lang/playground/blob/8126026daaf02f5a1901900b4ecf94e2da9811ff/backend/app.py#L13

allow requests from any origins.

We should restrict the origins to `localhost` (development) and `play.fortran-lang.org` (production).

@ashirrwad do you think you can tackle this? I'm not as familiar with Flask.

**Comment by ashirrwad** [comment]
Yes, of course. Allowing cors for all sources helped make development simpler. I'll add rules for specific origins.
